rosdep/base.yaml

```diff
@@ -1373,6 +1373,12 @@ libdevil-dev:
     portage:
       packages: [media-libs/devil]
   ubuntu: [libdevil-dev]
+libdlib-dev: 
+  debian: [libdlib-dev]
+  gentoo:
+    portage:
+      packages: [sci-libs/dlib]
+  ubuntu: [libdlib-dev]
```

Comment:
This isn't available in trusty, so this will not work, please use the `'*'` syntax and explicitly disable it for trusty, see: https://github.com/ros-infrastructure/rep/blob/master/rep-0111.rst#allow-package_argument-to-be-explicitly-null

Reply:
Ah, thank you, changed.